iree/turbine/kernel/lang/global_symbols.py

```diff
@@ -1,16 +1,38 @@
 from .._support.indexing import index_symbol
+import sys
 
 # Global symbols used throughout the code.
 
 # Address spaces.
 GLOBAL_ADDRESS_SPACE = index_symbol("$GLOBAL_ADDRESS_SPACE")
 SHARED_ADDRESS_SPACE = index_symbol("$SHARED_ADDRESS_SPACE")
 
+
 # Distribution symbols.
 WORKGROUP_0 = index_symbol("$WG0")
 WORKGROUP_1 = index_symbol("$WG1")
 WORKGROUP_2 = index_symbol("$WG2")
 
+
+def create_additional_workgroup_symbols():
```

Comment:
Instead of having to call `create_additional_workgroup_symbols` we can just add symbols dynamically to the module from `get_workgroup_symbol` call

Reply:
sure, will modify